web-api/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.ts

```diff
@@ -73,15 +73,14 @@ export const addDocketEntryForPaymentStatus = ({ caseEntity, user }) => {
 };
 
 const addDocketEntries = ({ caseEntity }) => {
-  const initialDocumentTypesListRequiringDocketEntry = Object.values(
+  let initialDocumentTypesListRequiringDocketEntry = Object.values(
     INITIAL_DOCUMENT_TYPES_MAP,
   );
 
-  remove(
-    initialDocumentTypesListRequiringDocketEntry,
+  initialDocumentTypesListRequiringDocketEntry = initialDocumentTypesListRequiringDocketEntry.filter(
     doc =>
-      doc === INITIAL_DOCUMENT_TYPES.petition.documentType ||
-      doc === INITIAL_DOCUMENT_TYPES.stin.documentType,
+      doc !== INITIAL_DOCUMENT_TYPES.petition.documentType &&
+      doc !== INITIAL_DOCUMENT_TYPES.stin.documentType,
```

Comment:
```suggestion
  const initialDocumentTypesListRequiringDocketEntry = Object.values(INITIAL_DOCUMENT_TYPES_MAP).filter(
    doc =>
      doc !== INITIAL_DOCUMENT_TYPES.petition.documentType &&
      doc !== INITIAL_DOCUMENT_TYPES.stin.documentType,
  );
```
looks good, id suggest chaining all the functions together to look something like this